tests/test_introduction_to_classes.py

```diff
@@ -5,6 +5,11 @@
 from codecademy.introduction_to_classes import Fruit
 from codecademy.introduction_to_classes import ReturningCustomer
 from codecademy.introduction_to_classes import Triangle
+from codecademy.introduction_to_classes import Animal6
+from codecademy.introduction_to_classes import ShoppingCart10
+from codecademy.introduction_to_classes import Shape12, Triangle12, Equilateral12
+from codecademy.introduction_to_classes import Employee14, PartTimeEmployee14
```

Comment:
please chain imports from same module

Reply:
Dima, I do not understand what I did wrong?

Reply:
Nothing really wrong, just chain your imports  == put imorts from same module on 1 line, with comma as a delimiter

Reply:
and sort the imported classes, functions alphabetically